publications/joss/paper.md

```diff
@@ -141,17 +142,6 @@ An example of the synthetic and observational dataset is shown in the following
 
 
 
-# Details on the synthetic dataset
```

Comment:
It seems like this is a duplicate of the above section, so I have removed this one.